src/app/GithubLogo.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import React from "react";
+
+type Props = {
+  className: string,
+  altText: string,
+};
+
+export default function GithubLogo(props: Props) {
```

Comment:
Ah, interesting. Exporting these as separate files is totally fine with
me: they seem nice and self-contained. If you prefer to include them in
the `Page` module only, that’s also perfectly fine. (As you may have
noticed, we don’t maintain any “one component per file” rule.)

Reply:
I generally tend towards one-component-per-file, because I like being able to scan files for exactly what I'm looking for :)